src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,46 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int NUMBER_OF_ELEMENTS = 10;
+    private final K[] keyArray;
+    private final V[] valArray;
+    private byte size = 0;
+
+    public StorageImpl() {
+        this.keyArray = (K[])new Object[NUMBER_OF_ELEMENTS];
+        this.valArray = (V[]) new Object[NUMBER_OF_ELEMENTS];
```

Comment:
```suggestion
        keyArray = (K[])new Object[NUMBER_OF_ELEMENTS];
        valArray = (V[]) new Object[NUMBER_OF_ELEMENTS];
```
Can we avoid using downcasting here?
(sorry, I remember that I left this comment a previous time. I don’t know why it wasn’t saved.)